srcalc/src/main/java/gov/va/med/srcalc/domain/model/NumericalRange.java

```diff
@@ -55,6 +55,9 @@
      */
     NumericalRange()
     {
+        // Set invalid sentinel values to ensure clients call the setters.
+        fLowerBound = Float.NaN;
+        fUpperBound = Float.NaN;
```

Comment:
No, it will be an Exception. This constructor is package-private and only used by Hibernate. User-driven code uses the public constructor.